docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -1,5 +1,32 @@
 # This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
 
+## Caller of an async local function cannot expect any of the callee�s code to execute.
```

Comment:
nit: GitHub diff shows some unicode character as the apostrophe for "callee's". Is that intentional?